tests/robustness/main_test.go

```diff
@@ -124,7 +153,7 @@ func (s testScenario) run(ctx context.Context, t *testing.T, lg *zap.Logger, clu
 	maxRevisionChan := make(chan int64, 1)
 	g.Go(func() error {
 		defer close(maxRevisionChan)
-		operationReport = traffic.SimulateTraffic(ctx, t, lg, clus, s.profile, s.traffic, finishTraffic, baseTime, ids)
+		operationReport, retriableErr = traffic.SimulateTraffic(ctx, t, lg, clus, s.profile, s.traffic, finishTraffic, baseTime, ids)
```

Comment:
nit: I think considering that this is the only place we are setting `retriableErr`, this should be fine, but in the future if we set it in any of the other goroutines, that might be a race.

How about returning the err instead of `nil` in this goroutine and catching it as part of `g.Wait()` below?